custom-generators/LatGenerator.js

```diff
@@ -1,63 +1,50 @@
 function LatGenerator() {}
```

Comment:
You need to import chance and instanciate it to use it. Don't understand how it can work currently :smile: 

```js
var chance = require("chance").Chance();

//...

chance.latitude(...);
```